database/types.go

```diff
@@ -19,7 +19,7 @@ func NewNullInt64(i int64) sql.NullInt64 {
 func NewNullString(s string) sql.NullString {
 	return sql.NullString{
 		String: s,
-		Valid:  true,
+		Valid:  s != "",
```

Comment:
is it important for this to change? the change means that an empty string will be saved as `null` in the DB, which I don't think is correct. i.e. an empty string should still be a valid string.

Reply:
yes, the new `block_value` field would be empty (due to simulation errors etc.) and inserting an empty string caused an error